assets/src/js/resistance-hub-campaign.js

```diff
@@ -1,5 +1,14 @@
 const {__} = wp.i18n;
 
+// Reorder completed posts in the Actions List block on the Resistance Hub template.
+const reorderCompletedPosts = () => {
+  [...document.querySelectorAll('.actions-list .wp-block-post-template > li:not(.completed)')]
+    .reverse()
+    .forEach(node => {
+      node.parentNode.insertBefore(node, node.parentNode.firstChild);
```

Comment:
Using `insertBefore` instead of `innerHTML` to update the original parent element